api/config.py

```diff
@@ -212,6 +223,8 @@ def initialize_db():
         upsert=True
     )
 
+    log_db.access_log.create_index('origin.id')
```

Comment:
Are you certain adding more indexes is a bad idea? If mongo will truly be temporary then it shouldn't be a problem. OTOH, if ya'll have already decided this then let me know :)